ci/requirements.freethreading.in

```diff
@@ -0,0 +1,11 @@
+# This is a dependency of pytest on Windows but isn't picked up by pip-compile.
+atomicwrites
+cibuildwheel
+#cffi
```

Comment:
given that we're disabling cffi with setup.py maybe it makes sense to uncomment this line.
probably less headaches in the future.

IIRC this was the only difference, so these special requirements files for free threading can be removed.
(I'm on my phone, sorry I can't check this right now.)